demo/core_33_$.pages.js

```diff
@@ -106,13 +106,23 @@ $.pages.home.jssp = [
   '  <h1>',
   '    <svg xmlns="http://www.w3.org/2000/svg" width="95px" height="100px">',
   '      <path',
-  '         d="m 1,90 0,-25 10,0 0,-8 9,0 0,-5 8,0 0,5 2,0 0,33 3,0 0,-13 2,0 0,-7 2,0 0,-32 1,0 0,-7 1,0 0,-6 2,0 0,-1 1,0 0,-12 1,0 0,-12 0,12 1,0 0,12 1,0 0,1 2,0 0,6 1,0 0,7 1,0 0,32 2,0 0,7 2,0 0,13 4,0 0,-32 5,0 0,-5 6,0 0,5 3,0 0,32 5,0 0,-42 3,0 0,-9 7,0 0,-2 5,0 0,2 2,0 0,51"',
+  '         d="m 1,90 0,-25 10,0 0,-8 9,0 0,-5 8,0 0,5 2,0 0,33 3,0 0,-13 2,0 0,-7 2,0 0,-32 1,0 0,-7 1,0 0,-6 2,0 0,-1 1,0 0,-12 1,0 0,-11 0,11 1,0 0,12 1,0 0,1 2,0 0,6 1,0 0,7 1,0 0,32 2,0 0,7 2,0 0,13 4,0 0,-32 5,0 0,-5 6,0 0,5 3,0 0,32 5,0 0,-42 3,0 0,-9 7,0 0,-2 5,0 0,2 2,0 0,51"',
   '         style="fill: none; stroke: #000; stroke-width: 1.5;" />',
+  '      <rect id="light" x="42" y="0" height="2" width="2" style="fill: none;stroke: none;" />',
   '    </svg>',
   '    Code City',
   '  </h1>',
   '  <p id="tagline">A community of inquisitive programmers.</p>',
   '  <iframe src="/login"></iframe>',
+  '  <script>',
+  '  var light = document.getElementById(\'light\');',
+  '  var lightActive = false;',
+  '  function blink() {',
+  '    light.style.fill = lightActive ? \'#f00\' : \'none\';',
+  '    lightActive = !lightActive;',
+  '  }',
+  '  setInterval(blink, 1000)',
```

Comment:
Disappointed this is not 100% CSS.  Surely you could even do slow transitions, as if it were incandescent?